### DISCUSSION TITLE
Auto completions for gametest?

### DISCUSSION THREAD

**Discussion by jason7141** [bridge-core/editor#1020] [Q&A]
I'm trying to get auto completions to work for scripts but I cannot find out how. I'm trying to make a custom ui menu

**Comment by jason7141** [comment]
I am on mobile

**Comment by ewanpeebles** [comment]
I had this problem previously too. However, I finally got it working by **changing the version of the modules that you import.** The way you do this is first ensuring that you have the "Beta APIs" experiment enabled, then going into the manifest.json of the behavior pack, finding the dependencies section, and changing the version from 1.0.0-beta to something supported. A list of supported versions for the @minecraft/server module can be found [here](https://learn.microsoft.com/en-us/minecraft/creator/scriptapi/minecraft/server/minecraft-server?view=minecraft-bedrock-stable), and for @minecraft/server-ui the link is [here](https://learn.microsoft.com/en-us/minecraft/creator/scriptapi/minecraft/server-ui/minecraft-server-ui?view=minecraft-bedrock-stable).
I know I'm a bit late for this, but I hope this solves your problem!
